build/Targets/Versions.props

```diff
@@ -45,9 +45,8 @@
     <NuGetReleaseVersion>$(RoslynFileVersionBase)</NuGetReleaseVersion>
     <NuGetPerBuildPreReleaseVersion>$(NuGetPreReleaseVersion)-$(BuildNumberFiveDigitDateStamp)-$(BuildNumberBuildOfTheDayPadded)</NuGetPerBuildPreReleaseVersion>
     <NuGetPackageRoot Condition="'$(NuGetPackageRoot)' == ''">$(NUGET_PACKAGES)</NuGetPackageRoot> <!-- Respect environment variable if set -->
```

Comment:
I think this variable is going to get pulled in regardless by the NuGet targets so if it doesn't have a slash we may not have any say in the matter. Plus, this env variable isn't just supposed to be for us, it's supposed to be the "official" way to override the NuGet package location, so I'd prefer to touch it as little as possible, with the expectation that we do no more and no less than NuGet to respect it.